Sources/SwiftTryCatch/SwiftTryCatch.h

```diff
@@ -0,0 +1,33 @@
+//
+//  SwiftTryCatch.h
+//
+//  Created by William Falcon on 10/10/14.
```

Comment:
This code was adapted from https://github.com/williamFalcon/SwiftTryCatch/

Reply:
Nice, I’m glad we’re attributing the original author.